### ISSUE TITLE
Build a great protocol-editor

### ISSUE THREAD

**Issue by marc-portier** [karel1980/scratchnet#13]
Writing protocol-specs should become as easy as snapping blocks together in scratch.
Let's collect some ideas and designs!

**Comment by karel1980** [comment]
I would make everything editor-related to 1.1

**Comment by marc-portier** [comment]
yeah - surely "later"  - allthough a textarea-edit inclusion in 1.0 would be nice
